src/octoprint/util/__init__.py

```diff
@@ -11,6 +11,7 @@
 
 import os
 import traceback
+import six
```

Comment:
Hmm... I've so far avoided using `six` (even though it is a sub dependency due to I think flask and werkzeug) and instead used [python future](https://python-future.org/) as compatibility layer. I think for consistency reasons it would probably be better to stick to that and instead of replacing `(str, unicode)` with six.string_types replace it with `basestring` imported from `past.builtins`.